packages/smooth_app/lib/pages/product/common/product_list_page.dart

```diff
@@ -195,7 +195,17 @@ class _ProductListPageState extends State<ProductListPage> {
                   );
                   if (dismissible) {
                     return Dismissible(
-                      background: Container(color: colorScheme.background),
+                      //Delete Animation Added
+                      direction: DismissDirection.endToStart,
```

Comment:
is the end to start behavior intended here, i guess it will limit the swipe feature to just one side